src/AkismetSpamProtector.php

```diff
@@ -66,7 +74,7 @@ class AkismetSpamProtector implements SpamProtector
     
```

Comment:
The code sniffer is complaining about the preceeding line (73) in that it begins with an underscore. Which is good, because otherwise I probably wouldn't have noticed that the `Readme.md` notes that the configuration value is `api_key`, not `_api_key`. This will need altering too.

Reply:
That is to say, please remove the underscore ;)

Reply:
as above this comment is closed by the newer changes, effectively retaining the functionality on an instance basis rather than a static one, which is nicer.